libs/feature/record/src/lib/record-metadata/record-metadata.component.css

```diff
@@ -2,6 +2,7 @@
 ::ng-deep .mat-tab-link {
   font-family: var(--font-family-main);
   letter-spacing: 0.88px;
+  color: var(--color-primary-lighter);
```

Comment:
there is a TW class for that, isn't it ?

Reply:
`text-primary-lighter` i think